### PR TITLE
ascanrulesBeta: add alert links to website help pages

### PR DESCRIPTION
**Pull request by thedudeontitan**
## Overview
added alert links to website help pages
fixes part of [zaproxy/zaproxy/issues/8189](https://github.com/zaproxy/zaproxy/issues/8189).

## Related Issues
[zaproxy/zaproxy/issues/8189](https://github.com/zaproxy/zaproxy/issues/8189)

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

**Comment by thedudeontitan** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by thedudeontitan** [comment]
recheck

**Review by kingthorin — DISMISSED** [review]

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/CommonActiveScanRuleBetaInfo.java
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/CommonActiveScanRuleBetaInfo.java:1]: You can drop Beta from the interface name, that'll make rule promotions easier

File: addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/CommonActiveScanRuleBetaInfo.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+package org.zaproxy.zap.extension.ascanrulesBeta;
+
+public interface CommonActiveScanRuleBetaInfo {
+
+    public int getPluginId();
+
+    public default String getHelpLink() {
+        return "https://www.zaproxy.org/docs/desktop/addons/active-scan-rules-beta/#id-"
+                + getPluginId();
```
Comment (kingthorin) [inline addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/CommonActiveScanRuleBetaInfo.java:28]: In the active scan rules it's `getid()`

**Comment by thedudeontitan** [comment]
I have made the requested changes and committed them also i have squashed the commits, please review

File: addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html [hunk 2]
```diff
@@ -10,29 +10,37 @@
...
 Checks if the web server is configured to allow Cross Domain access, from a malicious
 third party service, for instance. Currently checks for wildcards in Adobe's crossdomain.xml, and in
 SilverLight's clientaccesspolicy.xml. 
 <p>
 Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/CrossDomainScanRule.java">CrossDomainScanRule.java</a>
+<br>
+Alert ID: <a href="https://www.zaproxy.org/docs/alerts/20016/">20016</a>.
 
-<H2>CSRF Token</H2>
+<H2 id="id-10202">CSRF Token</H2>
```
Comment (thc202) [inline addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html:43]: Should be `20012`.

Comment (thedudeontitan) [reply addOns/ascanrulesBeta/src/main/javahelp/org/zaproxy/zap/extension/ascanrulesBeta/resources/help/contents/ascanbeta.html]: Updated the ID

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]
Thanks

**Pull request merged**